doc/source/file_formats.rst

```diff
@@ -46,9 +46,48 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
+The soma can be represented in one of the following formats: 
+
+TypeA: One point soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by its center and a radius that corresponds to the sphere
+that preserves the surface area of the soma. The center of the soma has as parent ID -1 
+and all the initial points of the neuronal trees are connected to the soma. 
+
+TypeB: Three point soma
```

Comment:
Hi @rbakker, I'd like to confirm the following points with you:

1) Is the three point soma in SWC format expected to have the following three points:
- center of the soma (x, y, z)
- translation in the y-coordinate (x, y + R, z), connected to the center
- translation in the y-coordinate (x, y - R, z), connected to the center

2) Is the multiple point soma in SWC expected to have:
- the center of the soma as the first point
- are the rest of the points connected to the center? 

Thanks a lot,
Lida